operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -190,13 +190,15 @@ type PerNodeSpec struct {
 	TaintToleration *TaintTolerationPolicy `json:"taintToleration,omitempty"`
 }
 
-// CollectionMethod defines the method of collection used by collector. Options are 'EBPF', 'KernelModule' or 'None'.
-// +kubebuilder:validation:Enum=EBPF;KernelModule;NoCollection
+// CollectionMethod defines the method of collection used by collector. Options are 'EBPF', 'CORE_BPF', 'KernelModule' or 'None'.
+// +kubebuilder:validation:Enum=EBPF;CORE_BPF;KernelModule;NoCollection
 type CollectionMethod string
 
 const (
 	// CollectionEBPF means: use EBPF collection.
 	CollectionEBPF CollectionMethod = "EBPF"
+	// CollectionCoreBpf means: use CORE_BPF collection.
+	CollectionCoreBpf CollectionMethod = "CORE_BPF"
```

Comment:
Please also capitalize the const name.
```suggestion
	CollectionCOREBPF CollectionMethod = "CORE_BPF"
```